pallets/nfts/src/lib.rs

```diff
@@ -175,6 +178,12 @@ pub mod pallet {
 		#[pallet::constant]
 		type CollectionDeposit: Get<DepositBalanceOf<Self, I>>;
 
+		/// The basic amount of funds that must be reserved for a collection approval.
+		// Key: `sizeof((CollectionId, AccountId, AccountId))` bytes.
+		// Value: `sizeof((Option<BlockNumber>, Balance))` bytes.
+		#[pallet::constant]
+		type CollectionApprovalDeposit: Get<DepositBalanceOf<Self, I>>;
```

Comment:
Note: Used in

- do_approve_collection_transfer: https://github.com/r0gue-io/pop-node/pull/387/files#r1881288135